### PR TITLE
Video 4708 recording start stop UI updates

### PR DESCRIPTION
**Pull request by timmydoza**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-4708](https://issues.corp.twilio.com/browse/VIDEO-4708)

### Description

This PR makes some changes with respect to the user interface seen when a user starts and stops recordings.

Specifically:

- When a user is in a room when a recording is started, they will see a "Recording has started" snackbar.
- When a user is in a room when a recording is stopped, they will see a "Recording Complete" snackbar, with information about how to find the recordings in the Twilio Console.
- When a user joins a room where a recording is in progress, they will see a "Recording is in progress" snackbar.

While in a room:
recording

Joining a room where recording is in progress:
recording2

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

**Review by charliesantos — APPROVED** [review]

File: src/components/MenuBar/Menu/Menu.test.tsx [hunk 1]
```diff
@@ -71,20 +76,11 @@ describe('the Menu component', () => {
         expect(getByText('Start Recording')).toBeTruthy();
       });
 
-      it('should open a dialog box when clicked', () => {
+      it('should correctly update recording rules and display the snackbar when the user clicks on the Start Recording button', () => {
         const { getByText } = render(<Menu />);
         fireEvent.click(getByText('More'));
 
         fireEvent.click(getByText('Start Recording'));
```
Comment (olipyskoty) [inline src/components/MenuBar/Menu/Menu.test.tsx:83]: just a question: what made you choose to use react testing library as opposed to enzyme ` .simulate('click');` like on line 133 below?

File: src/components/MenuBar/Menu/Menu.test.tsx [hunk 2]
```diff
@@ -143,7 +129,7 @@ describe('the Menu component', () => {
       expect(wrapper.find(DeviceSelectionDialog).prop('open')).toBe(false);
       wrapper
         .find(MenuItem)
-        .at(0)
+        .at(1)
         .simulate('click');
```
Comment (olipyskoty) [inline src/components/MenuBar/Menu/Menu.test.tsx:133]: see question above 🙂

File: src/components/RecordingNotifications/RecordingNoticifaction.test.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    ).toMatchInlineSnapshot(`"Recording is in progress."`);
+  });
+
+  it('should display "Recording Started" Snackbar when a recording is started after a user joins a room', () => {
+    const wrapper = mount(<RecordingNotifications />);
+    mockUseIsRecording.mockImplementation(() => false);
+    wrapper.setProps({}); // Set value of prevIsRecording.current
+
+    mockUseIsRecording.mockImplementation(() => true);
+    wrapper.setProps({}); // Set value of prevIsRecording.current
```
Comment (olipyskoty) [inline src/components/RecordingNotifications/RecordingNoticifaction.test.tsx:42]: i wasn't familiar with the `setProps()` method, i will remember this for when i need something to re-render in a test 🎉

**Review by olipyskoty — APPROVED** [review]
this looks good to me. just a couple of questions for my own learning!

**Pull request merged**